tmol/chemical/restypes.py

```diff
@@ -259,6 +262,75 @@ def _setup_ordered_torsions(self):
                 )
         return ordered_torsions
 
+    @property
+    def n_torsions(self):
+        return self.ordered_torsions.shape[0]
+
+    is_torsion_mc: numpy.ndarray = attr.ib()
+
+    @is_torsion_mc.default
+    def _setup_is_torsion_mc(self):
+        # A torsion is a "main chain" torsion if all of its atoms are
```

Comment:
Mainchain atoms are a property now?

What exactly depends on this?  is it just atom tree building?